src/bank/db/Customer.java

```diff
@@ -117,6 +124,8 @@ public String toString() {
             email +
             ", branch=" +
             branch +
+            // ", Admin_Stat=" +
+            // adminStatus +
```

Comment:
```suggestion
            ", adminStatus=" +
            adminStatus +
```
But that would mean you'd need to update all tests... Yeah...